setup.py

```diff
@@ -164,9 +164,9 @@ def run(self):
 
 
 setup(
-    name='azure-functions-python',
+    name='azure-functions-worker',
```

Comment:
rename to match the package azure.functions_worker

Reply:
Python package names customarily have dots and underscores replaced by dashes.  See, for example, https://pypi.org/project/azure-mgmt-storage/